recipes/pcre/all/conanfile.py

```diff
@@ -39,7 +40,8 @@ class PCREConan(ConanFile):
         "with_zlib": True,
         "with_jit": False,
         "with_utf": False,
-        "with_unicode_properties": False
+        "with_unicode_properties": False,
+        "with_stack_for_recursion": True
```

Comment:
I think the default should be `False`.
See https://github.com/LuaDist/pcre/blob/2b08a801c53f67b1d71fb83263fab819db78f886/CMakeLists.txt#L98
```suggestion
        "with_stack_for_recursion": False,
```

Reply:
CMake option is NO_RECURSE while this recipe option is recurse actually, so it's fine.

Reply:
No it shouldn't.
With_stack_for_recursion at True, it puts "PCRE_NO_RECURSE" at OFF, and vice-versa.
PCRE_NO_RECURSE = **not** with_stack_for_recursion

When building with autotools, the default option is stack for recursion enabled (ie PCRE_NO_RECURSE at OFF).

Reply:
Haha, thanks!
You are both right!